scripts/us_bls/jolts/bls_jolts.py

```diff
@@ -0,0 +1,285 @@
+
+  Returns:
+    jolts_df: A dataframe representing the 6 job data categories by industry, year, and seasonal adjustment
+    schema_mapping: A list of tuples that contains information for each data set 
+  """
+  
+  # Download series descriptions (used for seasonally adjusted status and industry code)
+  expected_series_columns = ['series_id','seasonal', 'industry_code', 'region_code', 'dataelement_code', 'ratelevel_code', 'footnote_codes','begin_year', 'begin_period', 'end_year', 'end_period']
+  series_desc = web_to_pandas("https://download.bls.gov/pub/time.series/jt/jt.series", sep="\t", expected_cols=expected_series_columns)
+  series_desc = series_desc.set_index("series_id")
+
+  # Download various series datapoints -- find more info here https://download.bls.gov/pub/time.series/jt/jt.txt
+  expected_job_columns = ['series_id', 'year', 'period', 'value', 'footnote_codes']
+  job_openings = web_to_pandas("https://download.bls.gov/pub/time.series/jt/jt.data.2.JobOpenings", sep="\t", expected_cols=expected_job_columns)
+  job_hires = web_to_pandas("https://download.bls.gov/pub/time.series/jt/jt.data.3.Hires", sep="\t", expected_cols=expected_job_columns)
+  total_seps = web_to_pandas("https://download.bls.gov/pub/time.series/jt/jt.data.4.TotalSeparations", sep="\t", expected_cols=expected_job_columns)
+  total_quits = web_to_pandas("https://download.bls.gov/pub/time.series/jt/jt.data.5.Quits", sep="\t", expected_cols=expected_job_columns)
+  total_layoffs = web_to_pandas("https://download.bls.gov/pub/time.series/jt/jt.data.6.LayoffsDischarges", sep="\t", expected_cols=expected_job_columns)
+  total_other_seps = web_to_pandas("https://download.bls.gov/pub/time.series/jt/jt.data.7.OtherSeparations", sep="\t", expected_cols=expected_job_columns)
+
+  # Additional information about each dataframe
+  # Tuple Format: Statistical Variable name, Stat Var population, Stat Var Job Change Type If Relevant, Dataframe for Stat Var 
+  schema_mapping = [
+    ("NumJobOpening", "schema:JobPosting", "", job_openings),
+    ("NumJobHire", "dcs:BLSWorker", "Hire", job_hires),
+    ("NumSeparation", "dcs:BLSWorker", "Separation", total_seps),
+    ("NumVoluntarySeparation", "dcs:BLSWorker", "VoluntarySeparation", total_quits),
+    ("NumInvoluntarySeparation", "dcs:BLSWorker", "InvoluntarySeparation", total_layoffs),
+    ("NumOtherSeparation", "dcs:BLSWorker", "OtherSeparation", total_other_seps),
+  ]
+
+  # Combine datasets into a single dataframe including origin of data
+  jolts_df = pd.DataFrame()
+  columns_to_keep = ['series_id', 'year', 'period', 'value', 'footnote_codes']
+
+  for schema_name, population_type, job_change_event, df in schema_mapping:
+    df = df.loc[:, columns_to_keep]
+    df['statistical_variable'] = schema_name
+    df['job_change_event'] = job_change_event
+    df['population_type'] = population_type
+    jolts_df = jolts_df.append(df)
+
+  # Drop non-monthly data
```

Comment:
```suggestion
  # Drop non-monthly data.
```

And add periods at the end of other imperative sentences in comments please.